lib/livebook_web/router.ex

```diff
@@ -83,13 +83,25 @@ defmodule LivebookWeb.Router do
       live "/hub/:id/secrets/edit/:secret_name", Hub.EditLive, :edit_secret, as: :hub
       live "/hub/:id/file-systems/new", Hub.EditLive, :new_file_system, as: :hub
       live "/hub/:id/file-systems/edit/:file_system_id", Hub.EditLive, :edit_file_system, as: :hub
-      live "/hub/:id/deployment-groups/new", Hub.EditLive, :new_deployment_group, as: :hub
+
+      live "/hub/:id/deployment-groups/new", Hub.Teams.DeploymentGroupLive, :new_deployment_group,
+        as: :hub
 
       live "/hub/:id/deployment-groups/edit/:deployment_group_id",
-           Hub.EditLive,
+           Hub.Teams.DeploymentGroupLive,
            :edit_deployment_group,
            as: :hub
 
+      live "/hub/:id/deployment-groups/edit/:deployment_group_id/secrets/new",
+           Hub.Teams.DeploymentGroupLive,
+           :new_secret,
+           as: :hub
+
+      live "/hub/:id/deployment-groups/edit/:deployment_group_id/secrets/edit/:secret_name",
+           Hub.Teams.DeploymentGroupLive,
+           :edit_secret,
+           as: :hub
```

Comment:
Is there a reason we don't include this as another section on the team edit page?

Reply:
Each deployment group will have their own page, because we will keep on adding functionality to it (and because we can have several deployment groups!)